### PR TITLE
Fix more rubocop errors

### PR DESCRIPTION
**Pull request by toyhammered**

### CODE REVIEW

File: .pryrc [hunk 1]
```diff
@@ -6,6 +6,6 @@ if defined?(PryByebug)
 end
 
 # Hit Enter to repeat last command
-Pry::Commands.command /^$/, 'repeat last command' do
+Pry::Commands.command (/^$/), 'repeat last command' do
```
Comment (NuckChorris) [inline .pryrc]: I... don't think this does what you think it does????

File: app/controllers/sessions_controller.rb [hunk 2]
```diff
@@ -6,7 +6,7 @@ def new
   def create
     return render status: 403, text: 'Token missing' if params[:token].blank?
     token = Doorkeeper::AccessToken.by_token(params[:token])
-    is_admin = token.resource_owner.roles.where(name: 'admin').count > 0
+    is_admin = token.resource_owner.roles.where(name: 'admin').positive?
```
Comment (NuckChorris) [inline app/controllers/sessions_controller.rb]: It didn't tell you to change `.count > 0`, it told you to change `> 0`

Anyways, the better option is `token.resource_owner.roles.where(name: 'admin').exists?`

File: app/models/block.rb [hunk 3]
```diff
@@ -29,15 +29,15 @@ class Block < ApplicationRecord
...
+    errors.add(:blocked, 'cannot be an admin') if blocked.has_role?(:admin)
+    errors.add(:blocked, 'cannot be a moderator') if blocked.title == 'Mod'
   end
 
   def self.hidden_for(user)
     return [] if user.nil?
     user = user.id if user.respond_to?(:id)
-    Block.where('user_id = ? or blocked_id = ?', *[user]*2).
-          pluck(:blocked_id, :user_id).flatten.uniq - [user]
+    Block.where('user_id = ? or blocked_id = ?', *[user] * 2)
```
Comment (NuckChorris) [inline app/models/block.rb]: prolly actually best to just do `.where('user_id = ? or blocked_id = ?', user, user)` instead of my stupid cleverness

**Review by NuckChorris — CHANGES_REQUESTED** [review]

File: app/controllers/sessions_controller.rb (hunk 2 above)
Comment (toyhammered) [reply app/controllers/sessions_controller.rb]: oh jeez good catch, yeah i should have just removed the `> 0` but will change to what you wrote above.

File: app/models/block.rb (hunk 3 above)
Comment (toyhammered) [reply app/models/block.rb]: (I was trying to figure out the meaning behind this, thanks for explaining)

Comment (NuckChorris) [reply app/models/block.rb]: Yeah it builds an array of two `user` (multiplies `[user]` times `2`) and then uses the splat operator (`*`) on it to make those into separate parameters.  It's clever, but it's also really really hard to read and I should never have done it

File: .pryrc (hunk 1 above)
Comment (NuckChorris) [reply .pryrc]: I think you mean `Pry::Comments.command(/^$/, 'repeat last command') do`, to be clear

Comment (toyhammered) [reply .pryrc]: yeah.. I think you are right.

File: app/models/block.rb (hunk 3 above)
Comment (toyhammered) [reply app/models/block.rb]: I mean I'll give you credit the idea is A+++ but other than that...

File: .pryrc (hunk 1 above)
Comment (toyhammered) [reply .pryrc]: well technically what I did is "right" (I think my stupidity beat rubocop). But anyways, changed it to be more correct.

**Pull request merged**